src/VideoSection/VideoSection.js

```diff
@@ -32,7 +34,15 @@ class VideoSection extends Component {
   componentDidMount() {
     fetchMovieVideos(this.state.movieID)
       .then(videoData => this.setState({ videos: videoData }))
-      .catch(err => this.setState({ error: err.message}))
+      .catch(err => this.setState({ error: this.handleErrorResponse(err.message)}))
+  }
+
+  handleErrorResponse(error) {
+    if (error < 500) {
+      return "Sorry, we couldn't find the videos for this movie."
+    } else {
+      return "Something went wrong. Please try again later"
+    }
```

Comment:
I wonder if some of this should live in the APICalls file. Not a big deal either way.